### PR TITLE
 Add a basic readme and usage examples

### LINKED ISSUE #2: Adding a minimal config example
Hey,

Thank you for bringing ctranslate2 to Triton! I am currently working on a project for a course using ctranslate2 and would like to host the model on Triton.

I got it up with Triton, but I am struggling to find the correct config.

Would it be possible to add a minimal example for a model?

Thank you!

### PR DESCRIPTION
**Pull request by HennerM**
This is a first step in documenting how to use the backend. 

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  {
+    name: "OUTPUT_IDS"
+    data_type: TYPE_INT32
+    dims: [ -1 ]
+  }
+]
+
+instance_group [{ kind: KIND_GPU, count: 1 }] # use KIND_CPU for CPU inference
+dynamic_batching {
+  max_queue_delay_microseconds: 5000 # can be tuned baesd on latency requirements
```
Comment (cdockes) [inline README.md]:
```suggestion
  max_queue_delay_microseconds: 5000 # can be tuned based on latency requirements
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    logging.basicConfig(level=logging.INFO)
+    asyncio.run(main())
+
+```
+
+You can try this by saving the script to `client.py` and running `echo "How is your day going?" | python3 client.py`.
+
+## Additional model config parameters:
+
+Currently we support a subset of decoding parameters to be set in the config, and an additional custom `max_decoding_length_multiple`:
```
Comment (cdockes) [inline README.md]: Maybe a sentence to explain what `max_decoding_length_multiple` does

File: README.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    value {
+      string_value: "2" 
+    }
+  },
+    {
+    key: "beam_size"
+    value {
+      string_value: "4" # needs to be string, since Triton does not support integers
+    }
+  }
```
Comment (cdockes) [inline README.md:135]: Also support repetition penalty now

File: examples/model_repo/facebook_m2m100_1.2B/config.pbtxt [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  {
+    name: "INPUT_IDS"
+    data_type: TYPE_INT32
+    dims: [ -1 ]
+    allow_ragged_batch: true
+  }
+]
+input [
+  {
+    name: "TARGET_PREFIX"
```
Comment (cdockes) [inline examples/model_repo/facebook_m2m100_1.2B/config.pbtxt:14]: Can we explain what this is used for in the readme?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+
+## Sending requests
+
+The backend expects token IDs as input and output, which need to be INT32 or bigger data-types. This might include beginning-of-sentence and end-of-sentence token, depending on the model. 
+No padding tokens need to be added though, the backend is taking care of padding and batching.
+
+You can use the offical Triton clients to make requests, both HTTP and gRPC protocols are supported. Here is an example for the opus-mt-en-de model in Python, using the async grpc client:
+
+```python
```
Comment (cdockes) [inline README.md]: Could we just point to examples/client.py?

**Review by cdockes — APPROVED** [review]
Very helpful!

File: README.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+Make sure to have [cmake](https://cmake.org) installed on your system.
+
+1. Build and install CTranslate2: [https://opennmt.net/CTranslate2/installation.html#compile-the-c-library](https://opennmt.net/CTranslate2/installation.html#compile-the-c-library)
+2. Build the backend 
+```bash
+mkdir build && cd build
+export BACKEND_INSTALL_DIR=$(pwd)/install
+cmake .. -DCMAKE_BUILD_TYPE=Release -DTRITON_ENABLE_GPU=1 -DCMAKE_INSTALL_PREFIX=$BACKEND_INSTALL_DIR
+```
```
Comment (aamir-s18) [inline README.md:18]: `make install` is missing here.

Comment (aamir-s18) [reply README.md:18]: Indeed very helpful, and great work!

File: README.md (hunk 1 above)
Comment (HennerM) [reply README.md]: Thanks for the catch

File: README.md (hunk 2 above)
Comment (HennerM) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (HennerM) [reply README.md]: Good idea. I wanted to keep it self contained first, but it grew too big

File: README.md (hunk 6 above)
Comment (HennerM) [reply README.md:18]: Thanks

**Pull request merged**